ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/inventory/host_vars/duplicate-interface-l3-edge.yml

```diff
@@ -16,14 +16,19 @@ l3_edge:
       interfaces: [ethernet1, ethernet1]
       id: 1
       profile: profile1
+      as: [65000, 65002]
+      ip: [192.168.0.2/31, 192.168.0.3/31]
 
     - nodes: [duplicate-interface-l3-edge, peer2]
       # duplicate interface for duplicate-interface-l3-edge
       interfaces: [ethernet1, ethernet2]
       id: 2
       profile: profile2
+      as: [65000, 65002]
+      ip: [192.168.0.2/31, 192.168.0.3/31]
 
 expected_error_message: >-
-  Found duplicate objects with conflicting data while generating configuration for Ethernet Interfaces defined under l3_edge p2p_link.
-  {'name': 'ethernet1', 'peer': 'peer2', 'peer_interface': 'ethernet2', 'description': 'P2P_peer2_ethernet2'} conflicts with
-  {'name': 'ethernet1', 'peer': 'peer1', 'peer_interface': 'ethernet1', 'description': 'P2P_peer1_ethernet1'}.
+  Found duplicate objects with conflicting data while generating configuration for Neighbors. {'ip_address': '192.168.0.3',
```

Comment:
This is detecting duplicate BGP neighbors now, where before it was checking for duplicate interfaces. Please revert and add another test for duplicate neighbors. Thanks.

Reply:
same for core-interface variant

Reply:
Fixed error message.